models/KINOVA_merge_without_floating_base_template.bioMod

```diff
@@ -564,7 +572,8 @@ segment part2
     rtinmatrix 0
     rt 0   0   0  xyz -0.120 0 0 // ajouter rot
     rotations z
-    ranges 	-pi/6 2*pi/3 // 0 2*pi/3
+    //ranges 	-pi/6 2*pi/3 // 0 2*pi/3
+    ranges 0 9*pi/20
```

Comment:
same ?

Reply:
Not yet because it will be part of a PR with penalty